### ISSUE TITLE
bug(LiveAnnouncer): Repeated message is not announced

### ISSUE THREAD

**Issue by eitak** [angular/components#27602]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

LiveAnnouncer does not announce a message when it is the same as the last announced message 
(using MacOS VoiceOver + Chrome).

### Reproduction

Using MacOS VoiceOver + Chrome

StackBlitz link: https://stackblitz.com/edit/components-issue-1j77ug?file=src%2Fapp%2Fexample-component.ts

Steps to reproduce:
1. Turn on VoiceOver
2. Open StackBlitz link
3. Click "Announce Hello" button
      -> VoiceOver announces "Hello"
4.  Click "Announce Hello" button again
      -> Expected: VoiceOver announces "Hello"
      -> Actual: No announcement

### Expected Behavior

VoiceOver announces the message.

### Actual Behavior

VoiceOver did not announce the message.

### Environment

- Angular: 16.1.8
- CDK/Material: 16.1.7
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): macOS

**Comment by amysorto** [comment]
I am unable to reproduce this on my end when testing so I will be closing this issue.

If you still run into this problem, please create a new issue with a video and add your chrome and macOS versions to help debug further.
